content/en/logs/indexes/_index.md

```diff
@@ -78,6 +78,27 @@ Update or remove this quota at any time when editing the Index:
 
 **Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
 
+### Attribute based exclusion filter
+
+In addition to the default sampling on the raw log (exclusion percentage set on `all logs`), you can choose to sample on a specific attribute holding an ID, like `UserID`, `SessionID`, `RequestID`.
+
+For example, exclude 99% of `http.request_id` logs for `source:nginx`:
+
+{{< img src="logs/indexes/index_exclusion_on_attribute.png" alt="index attribute-based exclusion filter"  style="width:70%;">}}
+
+**Note**: 
+
+* for a non faceted attribute, you need to prefix it with the `@` character.
+
+* for low cardinality attributes, we suggest to define an exclusion query and sample on all matching logs.
+
+  For example to exclude 99% of successful Nginx HTTP requests, define the exclusion query `source:nginx @http.status_code:200` on 99% of `all logs`.
```

Comment:
This is not the example matching the note up there. 
I think what you want to write here is that better than excluding logs from 50% of status codes, you'd better explicitly exclude 100% logs from status code 201, 204, 403...

But status code is probably not the best example here : might be better to use something you'd actually want to sample on, like internal IPs ... or something like this. And at the end of the day, not sure we want to cast light on this.

Reply:
OK, replacing by IP.